.gitignore

```diff
@@ -59,3 +59,6 @@ config/overlays/temp
 
 # release-notes
 /releasenotes
+
+# jetbrains
+.idea/
```

Comment:
nit: this should be in a global gitignore, not a repo specific one

Reply:
Removed